voice-change-o-matic/scripts/app.js

```diff
@@ -76,34 +76,19 @@ function init() {
     return curve;
   }
 
-  // Grab audio track via XHR for convolver node
-  let soundSource;
-  ajaxRequest = new XMLHttpRequest();
-
-  ajaxRequest.open(
-    "GET",
-    "https://mdn.github.io/voice-change-o-matic/audio/concert-crowd.ogg",
-    true
-  );
-
-  ajaxRequest.responseType = "arraybuffer";
-
-  ajaxRequest.onload = function () {
-    const audioData = ajaxRequest.response;
-
-    audioCtx.decodeAudioData(
-      audioData,
-      function (buffer) {
-        soundSource = audioCtx.createBufferSource();
-        convolver.buffer = buffer;
-      },
-      function (e) {
-        console.log("Error with decoding audio data" + e.err);
-      }
+  // Grab audio track via fetch() for convolver node
+  try {
+    const response = await fetch(
+      "https://mdn.github.io/voice-change-o-matic/audio/concert-crowd.ogg"
     );
-  };
-
-  ajaxRequest.send();
+    const arrayBuffer = await response.arrayBuffer();
+    const decodedAudio = await audioCtx.decodeAudioData(arrayBuffer);
+    convolver.buffer = decodedAudio;
+  } catch (error) {
+    console.error(
+      `Unable to fetch the audio file: ${name} Error: ${err.message}`
+    );
+  }
 
   // Set up canvas context for visualizer
   const canvas = document.querySelector(".visualizer");
```

Comment:
Line 198 + 202: Remove useless `let barHeight;` and replace `barHeight = dataArrayAlt[i];` with `const barHeight = dataArrayAlt[i];`